ui/views_test.py

```diff
@@ -32,6 +35,27 @@ def test_program_liveness(self):
             status_code=200
         )
 
+    def test_unauthenticated_user_redirect(self):
+        """Verify that an unauthenticated user can't visit '/dashboard'"""
+        response = self.client.get(reverse('ui-dashboard'))
+        self.assertRedirects(
+            response,
+            "/?next=/dashboard/"
```

Comment:
It looks like the path `/dashboard/` is used here, can you use `reverse` here too?